--- oracle-instantclient-12.1.0.2.0/instantclient_12_1/precomp/admin/pcscfg.cfg~	2014-07-07 11:08:58.000000000 +0300
+++ oracle-instantclient-12.1.0.2.0/instantclient_12_1/precomp/admin/pcscfg.cfg	2016-05-26 13:19:20.467720798 +0300
@@ -1,3 +1,3 @@
-sys_include=($ORACLE_HOME/sdk/include,/usr/include,/usr/lib/gcc/x86_64-redhat-linux/4.1.2/include,/usr/lib/gcc/x86_64-redhat-linux/4.4.7/include,/usr/lib/gcc/x86_64-redhat-linux/4.4.4/include,/usr/lib64/gcc/x86_64-suse-linux/4.1.2/include,/usr/lib64/gcc/x86_64-suse-linux/4.3/include)
+sys_include=(/usr/include/oracle/client,$ORACLE_HOME/sdk/include,/usr/include,@GCC_SYS_INC_DIR@)
 ltype=short
 define=__x86_64__
